libs/geography/testing/src/inmemory-backend/geography.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import {
+  InMemoryDbService,
+  RequestInfo
+} from 'angular-in-memory-web-api';
+
+import { DaffCountry } from '@daffodil/geography';
+
+import { DaffCountryFactory } from '../factories/public_api';
+
+/**
+ * An in-memory service that stubs out the backend services for getting countries.
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffInMemoryBackendGeographyService implements InMemoryDbService {
```

Comment:
This needs a `get` method

Reply:
oh yeah I forgot that we always need explicit methods.

Reply:
changed